src/services/config.js

```diff
@@ -156,6 +159,26 @@ class ConfigData {
     // Trim whitespace if given a non-null string.
     return string ? string.trim() : string;
   }
+
+  addApiKeysToProfile(profile) {
+    if (profile) {
+      const credentials = this.getApiKeysByProfileID(profile.id);
+      if (credentials) {
+        profile.apiKey = credentials.apiKey;
+        profile.apiSecret = credentials.apiSecret;
+      }
+    }
+  }
+
+  getApiKeysByProfileID(profileId) {
+    if (this.profiles) {
+      const { profiles } = this;
+      if (profileId && profiles.hasOwnProperty(profileId)) {
+        return { apiKey: profiles[profileId].apiKey, apiSecret: profiles[profileId].apiSecret };
```

Comment:
Can simplify to `return profiles[profileId];`

Reply:
Updated accordingly.